packages/test/test-service-load/src/runner.ts

```diff
@@ -398,24 +432,59 @@ function scheduleContainerClose(
 						);
 						setTimeout(() => {
 							if (!container.closed) {
-								container.close();
+								const count = runConfig.random.integer(
+									stashedOpsMin,
+									stashedOpsMax,
+								);
+								if (count > 0) {
+									test.waitLoaded()
+										.then(() => {
+											// Synchronously generate changes and close. There should be zero chance these ops
+											// round-trip before closing.
+											test.generateChanges(count);
```

Comment:
this should be captured via a new mechanism, which schedules the container going offline.

Reply:
it looks like we already have a schedule that will go offline, i think that should be enough, we then just need to schedule closeAndGetPendingLocalState if the container has gone offline, and supports offline